### PR TITLE
Linux Installation Commands

### LINKED ISSUE #1: Local Setup Script for linux users
Right now,  script to setup TestAtScale locally is for Mac users. With some small changes in commands, that should be linux alternatives, we can have the setup ready for linux users as well.

### PR DESCRIPTION
**Pull request by adicoder95**
Added Linux command for local installation.
Closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,3 +34,104 @@ bash mac_deployer.sh
...
+```
+- And run script
+``` bash
+$ bash mac_deployer.sh 
+```
+- These steps are for linux ubuntu distribution, if you are user of any other linux distributions you may change some commands as per the distribution you have.
+
+
+
+#### For Windows Users:
```
Comment (saurabh-prakash) [inline README.md:85]: This is being dealt with separately. Please follow this [issue](https://github.com/LambdaTest/test-at-scale-scripts/issues/2) for more details

File: README.md [hunk 2]
```diff
@@ -34,3 +34,104 @@ bash mac_deployer.sh
...
+
+#### Linux(Ubuntu) Users:
+- Note: Dollar sign($) represents command, it isn't included in command. You must know the password of root.
+- First update system
+``` bash
+$ sudo apt-get update
+```
+- Requirements, you have to install recent version of GCC and glibc.
+```bash
+$ sudo apt-get install gcc gilbc
```
Comment (saurabh-prakash) [inline README.md:47]: Is this command correct?

**Review by saurabh-prakash — COMMENTED** [review]
Please test it locally on a Linux distribution and paste the output of `kubectl get po -A` in PR description.

File: README.md (hunk 2 above)
Comment (adicoder95) [reply README.md:47]: yes, Please run without $

**Comment by SparshKesari** [comment]
Hey @adicoder95, Can you please update

**Pull request closed without merging**